parlai/tasks/sst/agents.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+# Download and build the data if it does not exist.
+
+from parlai.core.teachers import DialogTeacher
+from .build import build
+import os
+
+
+class SSTTeacher(DialogTeacher):
+    def __init__(self, opt, shared=None):
+        # store datatype
+        self.dt = opt['datatype'].split(':')[0]
+
+        # store identifier for the teacher in the dialog
+        self.id = 'sst'
+
+        self.SST_LABELS = ['negative', 'positive']
+
+        self.label_path, opt['datafile'] = self._path(opt)
+
+        super().__init__(opt, shared)
```

Comment:
these are the lines i'm referring to in the above comment fyi

Reply:
Thank you, Emily, I've fixed it now.